src/box/schema.cc

```diff
@@ -69,7 +69,7 @@ struct rlist on_alter_func = RLIST_HEAD_INITIALIZER(on_alter_func);
 
 struct entity_access entity_access;
 
-uint32_t
+API_EXPORT uint32_t
```

Comment:
The documentation says that it only goes up while in fact it may wrap around. This is probably not critical, because a client is supposed to update the schema as soon as it changes, but perhaps we should turn it in `uint64_t` while we are at it to make its semantics clearer? We won't be able to do it in future, because it'd break the ABI.

Reply:
I agree: added a commit changing the type of schema version and everything dependening on it to `uint64_t`.